nucypher/characters/unlawful.py

```diff
@@ -76,10 +79,85 @@ def attach_transacting_key(cls, blockchain):
         try:
             password = INSECURE_DEVELOPMENT_PASSWORD
             blockchain.w3.provider.ethereum_tester.add_account(cls.fraud_key, password=password)
-        except (ValidationError, ):
+        except (ValidationError,):
             # check if Vlad's key is already on the keyring...
             if cls.fraud_address in blockchain.client.accounts:
                 return True
             else:
                 raise
         return True
+
+
+class Amonia(Alice):
```

Comment:
First time I realize this, but I think these `Unlawful` characters don't belong here. They belong to `tests`; they're only used there. 
Look at it this way: what's different, in essence, in these characters to some of the fixtures we have in `test/fixtures.py`?
I guess this just another iteration of the old discussion in #1005

Reply:
At the end of the day, it's true that these classes are only useful when testing either the codebase or the deployed network to verify that the attacks described are impossible.

However, I don't think that this makes these classes *test infrastructure*.

I think that one of the strongest assurances we make with the `unlawful` module is that these classes work (or really, that they *don't* work) on the live, deployed network.

Our codebase doesn't only tell the story of our network, it also tells a smaller but still important story of what's not possible.  

I think that we send the strongest possible signal about the way we have considered these attack surfaces by furnishing these classes up front in our regular codebase with complete, warranted confidence that these attacks won't work if used in the real world.